src/renderer/src/lib/State.ts

```diff
@@ -1,4 +1,4 @@
-export class State<T extends {}, K extends keyof T = keyof T> {
+export class State<T extends object, K extends keyof T = keyof T> {
```

Comment:
I feel like this should be a record instead but also this class doesn't actually seem to be used anywhere (and I don't see why you'd use it over a regular Map) so might as well delete the file lol